cilium/cmd/kernel_check.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/cilium/cilium/pkg/datapath/linux/probes"
+	"github.com/spf13/cobra"
+)
+
+var kernelCheckCmd = &cobra.Command{
+	Use:   "kernel-check",
+	Short: "Checks whether machine is supporting required kernel capabilities",
+	Run: func(cmd *cobra.Command, args []string) {
+		kernelCheck()
+	},
+}
+
+func kernelCheck() {
+	probeManager := probes.NewProbeManager()
+	w := tabwriter.NewWriter(os.Stdout, 30, 8, 0, ' ', 0)
+	fmt.Fprintf(w, "%s\t%s\t%s\n", "FEATURE", "SUPPORTED", "REQUIRED")
+	if err := probeManager.ExitKernelCheck(); err != nil {
+		fmt.Println("Error occurred: ", err)
```

Comment:
```suggestion
		fmt.Println("Error occurred:", err)
```
`fmt.Println()` adds a space between operands. Except that what we actually want here is to print to standard error output and to exit, so I'd even recommend:
```suggestion
		Fatalf("%s", err)
```